### DISCUSSION TITLE
Actually show the pricing publicly for pro version

### DISCUSSION THREAD

**Discussion by chrisbadley** [Smile-SA/elasticsuite#2919] [Ideas]
When ever I see "call for pricing" or "book a demo" instead of a price list I instantly become disinterested because it feels like the vendor is trying to hide how much it is, and therefore it must be extremely expensive.

Be honest with your pricing, be up front and show it publicly.

**Comment by romainruaud** [comment]
Hi Chris, that's actually a good question. 

We do think that clients don't want to buy a "price" but rather a product that brings value to their business.

In France we do have a proverb that says : "Ce qui est cher, c'est ce qui ne marche pas", which could be rephrased to : "Something can be considered as expensive if this does not leverage your business".

We are deeply convinced that the Pro version is a real ROI-booster for merchants and should not be seen only through the "price" glasses.

That's why we prefer to be able to demonstrate ourselves the value of the product through the demos we do to agencies and customers, rather than just display an always deceptive price list on our website. We all know how it's like : _Starting from XXX € / month_ and then you get into the sales funnel and at the end you realize that the starting price is totally uncorrelated with your real expense at the end of the year.

We also have the conviction that we've been **very** transparent for years, considering the fact that the open source version of Elasticsuite has always been free and readable for everyone. And considering the amount of websites using it nowadays, we consider that we are trusted by a lot of people, and that Elasticsuite has a real value on the market. If you trusted our free version for years, why would it be different for the pro version ? It's the same team, same product :)

Best regards

**Reply by chrisbadley** [reply]
I kind of understand the logic, but without being upfront you are going to waste alot of time dealing with smaller merchants who will never be able to get an ROI from the product simply due to lack of traffic and turnover.

If possible you should try and convert the product recommendation part of the software to a pricing model like Nosto and others do, where its percentage of sale, then you can be upfront as you like about the figure and it wont scare anyone away nor do you have to justify any initial or ongoing outlay to your customer. How you would track everything, I don't know, but I would like to maybe try the premium product for my clients, but the problem is.. how on earth can I begin to explain how they can get an ROI or even get them interested in the product without knowing the cost.

**Reply by romainruaud** [reply]
> I would like to maybe try the premium product for my clients, but the problem is.. how on earth can I begin to explain how they can get an ROI or even get them interested in the product without knowing the cost.

Just get in touch with us by mail at elasticsuite@smile.fr or through the website, we do have a partner program for agencies, we'll be able to give you the average pricing, the different packages we can propose, and you'll be able to know if this fits with your clients or not. Worst case scenario : you'll lose 30min of your time discussing with us.
